packages/webapp-libs/webapp-api-client/src/tests/factories/auth.ts

```diff
@@ -22,6 +22,12 @@ export const currentUserFactory = createFactory<CurrentUserType>(() => ({
         invitationAccepted: true,
         invitationToken: makeId(32),
         role: TenantUserRole.OWNER,
+        inviteeEmailAddress: 'invitee@example.org',
+        userId: makeId(32),
+        firstName: 'MembershipFirstName',
+        lastName: 'MembershipLastName',
+        userEmail: 'membership@example.org',
+        avatar: 'https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/316.jpg',
```

Comment:
This link does not work. Better to use sth like `https://example.com/avatar.jpg`?